api/types/image.go

```diff
@@ -0,0 +1,30 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+package types
+
+import (
+	dockertypes "github.com/docker/docker/api/types"
```

Comment:
I am not sure if we should use docker types.ImageSummary definition, since our API is not docker specific.

Reply:
I see. BTW, the `ImageClient` interface is using `types.ImageSummary`. Should we refactor these?

Reply:
I think so. We should refactor the ImageClient.

Reply:
Updated. PTAL.